doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -469,9 +469,9 @@ server side component installed to the Kubernetes Cluster (e.g.: Tiller).
 
 ## 3.3 CaaS Manager - Cluster Lifecycle Management
 
-In order to provision multiple Kubernetes clusters, which is a common scenario where workloads and network functions require dedicated, single-tenant, clusters, the the Reference
+In order to provision multiple Kubernetes clusters, which is a common scenario where workloads and network functions require dedicated, single-tenant, clusters, the Reference
 Architecture must provide support for a **CaaS Manager**: a component responsible for the Lifecycle Management of multiple Kubernetes Clusters.
```

Comment:
To address some concerns about "what" and not "how", should we change this to:
Architecture must support the Life-cycle Management of multiple Kubernetes Clusters.

The complete sentence would then read:
In order to provision multiple Kubernetes clusters, which is a common scenario where workloads and network functions require dedicated, single-tenant, clusters, the Reference Architecture must support the Life-cycle Management of multiple Kubernetes Clusters.